src/ops.jl

```diff
@@ -63,10 +67,10 @@ end
 macro define_broadcast(jl_op, tf_func)
```

Comment:
One day, and that day is not today,
we need to go and fix how we do broadcasting.
And use broadcasting styles fully.